test/test_katello/test_deb_tracer.py

```diff
@@ -0,0 +1,176 @@
+import unittest2 as unittest
```

Comment:
I am wondering: which features from `unittest2` do you need that aren't in regular `unittest`?